indico/modules/events/editing/models/editable.py

```diff
@@ -82,3 +82,13 @@ def __repr__(self):
     @locator_property
     def locator(self):
         return dict(self.contribution.locator, type=self.type.name)
+
+    def can_comment(self, user):
+        return self.editor == user or self.contribution.is_user_associated(user, check_abstract=True)
+
+    @property
+    def editors(self):
```

Comment:
I think we want to check only the `editor` assigned to the Editable, at least for anything that modifies data.

Not sure about creating new internal comments, I guess it'd make sense to let all editors (based on the not-yet-existing event permission, and the historical list of editors on the revision) create internal comments.

Reply:
@meluru opinions?